shock-server.cfg

```diff
@@ -0,0 +1,56 @@
+[Address]
+# IP and port for api
+# Note: use of port 80 may require root access
+# 0.0.0.0 will bind Shock to all IP's
+api-ip=0.0.0.0
+api-port=7044
+
+[Admin]
+email=kkeller@lbl.gov
+users=ciservices
+
+[Anonymous]
+# Controls an anonymous user's ability to read/write
+# values: true/false
+read=true
+write=true
+create-user=true
+
+[Auth]
+; for auth2
+globus_token_url=https://ci.kbase.us/services/auth/api/legacy/globus/goauth/token?grant_type=client_credentials
+globus_profile_url=https://ci.kbase.us/services/auth/api/legacy/globus/users
+
+
+[External]
+# api-url=https://ci.kbase.us/services/shock-api
+
+[Log]
+perf_log=false
+
+[Mongodb]
+# Mongodb configuration
+# Hostnames and ports hosts=host1[,host2:port,...,hostN]
+hosts=mongo
+database=shock
+user=
+password=
+gomaxprocs=4
+admin-email=sysadmins@kbase.us
```

Comment:
These admin directives look like they're in the wrong place (even though they are actually in the existing CI shock.cfg file).